health_monitor/src/driver_manager.cpp

```diff
@@ -194,47 +194,47 @@ namespace health_monitor
             else if((are_critical_drivers_operational_truck(time_now)=="s_1_l1_0_l2_1_g_1") || (are_critical_drivers_operational_truck(time_now)=="s_1_l1_1_l2_0_g_1"))
             {
             
-                alert.description = "Only one LIDAR, GPS and SSC is currently working";
+                alert.description = "One LIDAR Failed";
                 alert.type = cav_msgs::SystemAlert::CAUTION;
                 return alert;
             }
             else if((are_critical_drivers_operational_truck(time_now)=="s_1_l1_0_l2_1_g_0") || (are_critical_drivers_operational_truck(time_now)=="s_1_l1_1_l2_0_g_0"))
             {   
-                alert.description = "GPS stopped running but one LIDAR and SSC is still working";
+                alert.description = "GPS Failed";
                 alert.type = cav_msgs::SystemAlert::CAUTION;
                 return alert;
             }
             else if(are_critical_drivers_operational_truck(time_now)=="s_1_l1_0_l2_0_g_1")
             {
-                alert.description = "Both LIDARS are not running but GPS and SSC is still working";
+                alert.description = "Both LIDARS Failed";
                 alert.type = cav_msgs::SystemAlert::WARNING;
                 return alert;
             }
             else if(are_critical_drivers_operational_truck(time_now)=="s_1_l1_0_l2_0_g_0")
             {
-                alert.description = "All LIDARS and GPS stopped running, but SSC is working";
+                alert.description = "LIDARS and GPS Failed";
                 alert.type = cav_msgs::SystemAlert::FATAL;
                 return alert;
             }
             else if(are_critical_drivers_operational_truck(time_now)=="s_0")
             {
-                alert.description = "SSC stopped working";
+                alert.description = "SSC Failed";
                 alert.type = cav_msgs::SystemAlert::FATAL;
                 return alert;
             }
             else
             {
-                alert.description = "UNKNOWN";
+                alert.description = "Missing unknown essential driver";
```

Comment:
I feel like getting here is most likely a logic problem, not a driver/hw problem.  Therefore, a more appropriate message might be "Unknown problem assessing essential driver availability".